build-data-capturing-gradle-samples/README.md

```diff
@@ -0,0 +1,97 @@
+# Build Data Capturing Gradle Samples
+
+## Overview
+
+This directory contains samples demonstrating various ways to extend and customize a Build Scan with extra data using 
+tags, links, and custom values.
+
+To learn more, see the Gradle Enterprise documentation
+on [Extending build scans](https://docs.gradle.com/enterprise/gradle-plugin/#extending_build_scans). 
+
+### Capture Dependency Resolution
+
+[//]: # (todo figure out what this sample is doing)
```

Comment:
I struggled to understand what this sample was doing, how it could be useful, and how to create a scenario in which this sample would activate. I could use some help here!

Reply:
Good question, the idea of this script is to capture eager dependency resolution since this is a performance problem.

I'm not sure how much overlap this has with the current dependency resolution shown in the build scans now that build scans distinguish user initiated from Gradle initiated and tell you which configurations were resolved.